packages/kbn-user-profile-components/src/user_profile.ts

```diff
@@ -26,6 +26,11 @@ export interface UserProfile<D extends UserProfileData = UserProfileData> {
    */
   uid: string;
 
+  /**
+   * Indicates whether user profile is enabled or not.
+   */
+  enabled: boolean;
```

Comment:
__note:__ Covered the rationale here https://github.com/elastic/kibana/blob/ea7bc3619dddf4c4a4a2b8a71cbd3d9f29eac4b4/x-pack/plugins/security/server/user_profile/user_profile_service.ts#L182-L185

Reply:
Ah nice, thanks for this Oleg. I wrongly removed this prop from the interface thinking it wasn't used anymore.

Reply:
No worries, I reviewed this PR and thought that was fine to remove it too :see_no_evil: Only when migrating to the new Bulk Get API I realized that our consumers might need this field to recognize not active user profiles (e.g. in cases assignments).